src/sagemaker/network.py

```diff
@@ -20,24 +20,38 @@ class NetworkConfig(object):
     """Accepts network configuration parameters and provides a method to turn these parameters
     into a dictionary."""
 
-    def __init__(self, enable_network_isolation=False, security_group_ids=None, subnets=None):
+    def __init__(
+        self,
+        enable_network_isolation=False,
+        security_group_ids=None,
+        subnets=None,
+        encrypt_inter_container_traffic=None,
+    ):
         """Initialize a ``NetworkConfig`` instance. NetworkConfig accepts network configuration
         parameters and provides a method to turn these parameters into a dictionary.
 
         Args:
             enable_network_isolation (bool): Boolean that determines whether to enable
                 network isolation.
+            encrypt_inter_container_traffic (bool): Boolean that determines whether to
+                encrypt inter-container traffic.
```

Comment:
can you also add a sentence about what the default behavior is?

also, move this down to the end of the docstring so that the order matches the order of the parameters

Reply:
Done